src/main/java/org/alliancegenome/curation_api/services/helpers/gff3/Gff3AttributesHelper.java

```diff
@@ -0,0 +1,55 @@
+package org.alliancegenome.curation_api.services.helpers.gff3;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.alliancegenome.curation_api.enums.BackendBulkDataProvider;
+import org.alliancegenome.curation_api.model.ingest.dto.fms.Gff3DTO;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import jakarta.enterprise.context.RequestScoped;
+
+@RequestScoped
```

Comment:
With the case that is has only one static method the @RequestScoped can be removed.

Reply:
Removed